google/resource_attached_disk.go

```diff
@@ -0,0 +1,286 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	compute "google.golang.org/api/compute/v1"
+)
+
+func resourceAttachedDisk() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAttachedDiskCreate,
+		Read:   resourceAttachedDiskRead,
+		Update: resourceAttachedDiskUpdate, // TODO remove this method? make sure it's not called? etc
```

Comment:
Yeah you can just remove it and then set all the fields (except output-only ones) to `ForceNew`